user-guide/Advanced_Modules/Data_Aggregator/Data_Aggregator_settings.md

```diff
@@ -114,6 +114,48 @@ For example:
 
 The GQI queries themselves should be configured in separate JSON files. See [Configuring GQI queries for Data Aggregator](xref:Data_Aggregator_queries).
 
+### [From Data Aggregator 3.0.0 onwards](#tab/tabid-1)
```

Comment:
Do you need the *DataAPI* soft-launch option to use this? If yes, we should mention this below.